src/bors/handlers/trybuild.rs

```diff
@@ -301,7 +301,9 @@ fn auto_merge_commit_message(
 
 fn trying_build_comment(head_sha: &CommitSha, merge_sha: &CommitSha) -> Comment {
     Comment::new(format!(
-        ":hourglass: Trying commit {head_sha} with merge {merge_sha}…"
+        ":hourglass: Trying commit {head_sha} with merge {merge_sha}…
+
+To cancel the try build, run the command `try cancel`."
```

Comment:
The cancel command example should contain the name of the bot. It's not easily available now though, we would have to store the bot name into `BorsContext`, and then propagate the context all the way to the function that calls `trying_build_comment`, and `trying_build_comment` would then receive the bot name.